docs/source/glossary.rst

```diff
@@ -10,13 +10,13 @@ Glossary
         The configuration is a set of parameters that will define the level for each
         noise type in a ConfigTree object.
 
-    Forms
-        Forms are analogous to a dataset that will be noised and returned to the
+    Datasets
+        Datasets are analogous to a dataset that will be noised and returned to the
```

Comment:
Right, that's what I mentioned about there being kind of awkward translations now. I wasn't going to think too hard about that since we're all combing through docs this week but I'll fix this one anyway